bin/ohai

```diff
@@ -19,12 +19,6 @@
 # limitations under the License.
 #
 
-begin
-  require "rubygems"
-rescue LoadError
-  # must be debian! ;)
-  missing_rubygems = true
```

Comment:
we still check `missing_rubygems` below (line 34)

Reply:
🤕 Fixed up now